### PR TITLE
Add request form and Improve the code in mealDetails page

### LINKED ISSUE #34: Create meal request page, allow rendering of chef's past meals

### PR DESCRIPTION
**Pull request by obay-mardini**
closes #34

### CODE REVIEW

File: client/src/containers/MealDetails.js [hunk 1]
```diff
@@ -35,56 +35,57 @@ class MealDetails extends Component {
...
+                title={currentMeal.name}
+                subtitle={currentMeal.chef.username}
               />
             }
           >
             <img
-              src={this.props.meal.images}
-              alt="meal"
+              src={currentMeal.images}
+              alt="currentMeal"
```
Comment (gzeegz) [inline client/src/containers/MealDetails.js]: Let's change the `alt` text to be more descriptive. Perhaps `currentMeal.name`.

File: client/src/containers/NavBar.js [hunk 2]
```diff
@@ -28,10 +28,13 @@ class NavBar extends Component {
...
-          <li><Link to="/post-new-meal">New Meal</Link></li>
-          <li className={`nav-auth ${!user.user ? null : 'hidden'}`}>
-            <Link to="/signup">Signup</Link>
-          </li>
+          {user.role === 'chef'
+            ? <li><Link to="/post-new-meal">New Meal</Link></li>
+            : null}
+          {user.role === 'chef'
+            ? <li><Link to="/request-form">New Request</Link></li>
+            : null}
```
Comment (gzeegz) [inline client/src/containers/NavBar.js]: No need for ternary here. We can use `&&` operator instead.

**Review by gzeegz — CHANGES_REQUESTED** [review]
Looks good. Just need to make some small changes.

**Review by gzeegz — APPROVED** [review]

**Comment by rs214** [comment]
LGTM

**Pull request merged**